### PR TITLE
Search bar

### PR DESCRIPTION
**Pull request by hiroya210**
Added search bar at homepage
image
image

Added Components/_card.html.erb
Implemented card components on Home and Show page
Added CSS variable `$kondo` and `$kondo2` for color scheme
Made little changes on Navbar

### CODE REVIEW

File: app/views/kondos/index.html.erb [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-        <h5><%= kondo.summary %></h5>
-        <span><%= kondo.prefecture %></span>
-        <span><%= kondo.user.first_name %></span>
-        <span><%= kondo.price %></span>
-      </div>
-      <% end %>
-    </div>
+  <div class="row">
+    <% @kondos.each do |kondo| %>
+      <div class="col-xl-3 col-md-6 col-sm-12">
```
Comment (carlnoval) [inline app/views/kondos/index.html.erb:4]: @hiroya210 I like how the cards look man!

Played with this locally and I noticed that the centering becomes off when resizing the window. That said what I could recommend is to add the following classes on this div.

```html
<div class="col-xl-3 col-md-6 col-sm-12 d-flex justify-content-center">
```
or anywhere else that you'd find appropriate as long as the centering would be fine after resize

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+<div class="card-category d-flex flex-column justify-content-center align-items-center" 
+style="height: 100vh; background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.3)), url('<%= cl_image_path "https://res.cloudinary.com/djlvhfuba/image/upload/v1643110748/interior_ncfcmy.png", crop: :fill %>')">
+  <h1 style="font-size: 3.5rem; font-weight: bolder; color: white;">Make Your Home a Better Place</h1>
+  <%= render "components/searchBar" %>
 </div>
+<div class="container">
+  <h2>Interior Specialists near you</h2>
+    <div class="row justify-content-center">
+      <% @kondos.each do |kondo| %>
+        <div class="col-xl-3 col-md-6 col-sm-12">
```
Comment (carlnoval) [inline app/views/pages/home.html.erb:10]: Here too please, same with previous comment.
```html
<div class="col-xl-3 col-md-6 col-sm-12 d-flex justify-content-center">
```

**Review by carlnoval — COMMENTED** [review]
I'll leave a video clip on slack of what I mean.

File: app/views/kondos/index.html.erb (hunk 1 above)
Comment (carlnoval) [reply app/views/kondos/index.html.erb]: Thanks, its fixed now.

**Comment by carlnoval** [comment]
Just need to resolve conflicts now  and we can merge this 👍🏼

**Comment by hiroya210** [comment]
Hey team I already resolve the conflict for this one, please review and merge :)

**Pull request merged**